### ISSUE TITLE
Additional Speech Needed on Drum Kit Score Output

### ISSUE THREAD

**Issue by jmwmusic** [musescore/MuseScore#32802]
### Issue type

Accessibility issue (e.g. for keyboard-only or screen reader users)

### Description with steps to reproduce

Currently a blind user has no easy way to know what their percussion music looks like in print on the score.  For other instruments this added info is not required as as long as you know the clef you know the space/line position. However, as percussion scoring can vary this info is required.

This could be done in one of two ways (or both):
1.  Add the additional info of line/space, stem direction and note head shape at the end of the string of info spoken as you arrow along the score notes.  So for example "Bass Drum 1, crotchet, voice 2, bar 1 beat 1, space 1, filled note head, stem down".  As voices are announced maybe the stem down isn't necessary, but the line/space is the crucial missing bit and the X or circled heads.
2. These could be discovered by going to the Percussion panel and changing to Notation Preview instead of pad name.  See #32788.

### Supporting files, videos and screenshots

N/A

### In which versions of MuseScore Studio is this issue present?

4.7 beta

### Regression

I was unable to check

### Operating system

Windows 11 with NVDA

### Checklist

- [x] This report follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting bugs and issues
- [x] I have verified that this issue has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar issues
- [x] I have attached all requested files and information to this report
- [x] I have attempted to identify the root problem as concisely as possible, and have used minimal reproducible examples where possible
